src/pages/en/reference/directives-reference.md

```diff
@@ -4,7 +4,7 @@ title: Template Directives Reference
 i18nReady: true
 ---
 
-**Template directives** are a special kind of HTML attribute available inside of any Astro component template (`.astro` files).
+**Template directives** are a special kind of HTML attribute available inside of any Astro component template (`.astro` files), and in `.mdx` files.
```

Comment:
```suggestion
**Template directives** are a special kind of HTML attribute available inside of any Astro component template (`.astro` files), and some can also be used in `.mdx` files.
```
Do you think it works if we just say this @calebeby?

Reply:
Yeah 👍